kutt.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/bugsnag/microkit/clog"
+	"github.com/pkg/errors"
+	"github.com/spf13/viper"
+)
+
+type Client struct {
+	HTTPClient *http.Client
+	ApiKey     string
+	BaseURL    string
+	UserAgent  string
+}
+
+type SubmitParams struct {
+	URL      string `json:"target"`
+	ExpireIn string `json:"expire_in"`
+}
+
+func NewClient() *Client {
+	var cli Client
+	cli.ApiKey = viper.GetString("kutt_api_key")
+	cli.BaseURL = viper.GetString("kutt_base_url")
+	cli.UserAgent = "promalert/v1 (+https://github.com/bugsnag/promalert)"
+
```

Comment:
We should just define our own httpclient in the initializer instead of using the default http client:

```suggestion
  cli.HTTPClient = &http.Client{
  Timeout: time.Second * 10,
}
```